code/__defines/sound.dm

```diff
@@ -69,7 +69,9 @@
 // Restricted, military, or mercenary aligned locations like the armory, the merc ship/base, BSD, etc.
```

Comment:
Stuff in ambience tending towards background music (Yes I seriously listened to everything in ambience)
Chapel - [All]
Foreboding - 1, 2, 5, 6, 7
Ghostly - 1 (Although it might be ambient for the outpost when the winds are really high)
Highsec - 1, 2
Otherworldly - 3 (Enough sharp changes to stand out, though 1 and 2 might be good in the mines!)
Ruins - 2
Sif - 1
Space - Serithi, 1
Unholy - 1
ambispace
song_game
vaporwave